documentation/docs/specs/revocation_bitmap_2022.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Revocation Bitmap 2022 Specification
+sidebar_label: Revocation Bitmap 2022 Specification
+description: The specification for the embedded revocation bitmap.
+image: /img/Identity_icon.png
+keywords:
+  - DID
+  - specs
+  - specifications
+  - revocation
+  - bitmap
+---
+
+# Revocation Bitmap 2022
+
+## Abstract
+
+This specification describes an on-Tangle mechanism for publishing the revocation status of [Verifiable Credentials](../concepts/verifiable_credentials/overview) embedded in an issuer's DID document.
+
+## Introduction
+
+Revocation gives an issuer the capability to invalidate a credential they issued before its natural expiration date. To achieve this, issuers can embed an identifier in the `credentialStatus` field of a credential. Verifiers can then lookup that identifier in a separate list, to check whether the credential is still valid. This document specifies a mechanism of embedding such a list, in form of a bitmap, in the DID document of the issuer, where each bitmap index corresponds to a credential they have issued. This mechanism is space-efficient and enables a verifier to check a credential's status in a privacy-preserving manner and without requiring additional lookups or external resources.
+
+## Revocation Bitmap Concept
+
+The revocation status of a verifiable credential is expressed as a binary value. The issuer keeps a bitmap of indices corresponding to verifiable credentials they have issued. If the binary value of the index in the bitmap is 1 (one), the verifiable credential is revoked, if it is 0 (zero) it is not revoked.
+
+## Data Model
+
+### Revocation Bitmap Status
+
+For an issuer to enable verifiers to check the status of a verifiable credential, the [`credentialStatus`](https://www.w3.org/TR/vc-data-model/#status) property MUST be specified with the following properties:
+
+| Property | Description |
+| :--- | :--- |
+| `id` | The constraints on the `id` property are listed in the [Verifiable Credentials Data Model specification](https://www.w3.org/TR/vc-data-model/). The `id` MUST be a [DID URL](https://www.w3.org/TR/did-core/#did-url-syntax) that resolves to a [Revocation Bitmap Service](#revocation-bitmap-service) in the DID Document of the issuer. |
```

Comment:
I've asked for clarification on this [here](https://github.com/w3c-ccg/vc-status-list-2021/issues/31). Since we've released 0.6 already, I think we can merge the spec as-is and revise later if needed.